src/protocols/vnc/vnc.c

```diff
@@ -397,6 +397,33 @@ void* guac_vnc_client_thread(void* data) {
                 return NULL;
             }
 
+            /* Import the public key, if that is specified. */
+            if (settings->sftp_public_key != NULL) {
+
+                guac_client_log(client, GUAC_LOG_DEBUG,
+                        "Attempting public key import");
+
+                /* Attempt to read public key */
+                if (guac_common_ssh_user_import_public_key(vnc_client->sftp_user,
+                            settings->sftp_public_key)) {
+
+                     /* Public key import fails. */
+                         guac_client_abort(client,
```

Comment:
:upside_down_face:

Reply:
Can we say copy pasta?!

Fixed via rebase.